docs/source/intents.md

```diff
@@ -0,0 +1,40 @@
+# Intents Documentation
+
+This documentation provides information on the use of Intents to annotate data, including how they're applied to `OperationPlugin`'s.
+
+*If you are new to developing Xi-CAM operations,
+it is recommended that you follow the [operations development guide](operation-plugin.md) first.*
+
+For more general development resources, see the [Resources](resources.md) page.
+
+## What Is an Intent?
+
+In Xi-CAM, an `Intent` generally represents a descriptive annotation of how to visualize data.
+Basically, an `Intent` is a recipe for constructing a visualization.
+When an `Intent` is associated with an operation, Xi-CAM may use it to automatically generate plots as you adjust your workflow.
+An `Intent` can be embedded in a Databroker `BluesklyRun`, so they may be preserved along with derived (and also raw) data.
+
+### The Lifecycle of an Intent
+
+A `GUIPlugin` that supports intents will contain a `CanvasManager`.
```

Comment:
we might need to explain what a canvas manager is? depends on how detailed we want the documentation to be here

Reply:
separate document for more advanced components